modulefiles/module_base.hera

```diff
@@ -4,7 +4,8 @@
 ##
 
 # Modules availble through hpc-stack
-module use /scratch2/NCEPDEV/nwprod/hpc-stack/libs/hpc-stack/modulefiles/stack
+# module use /scratch2/NCEPDEV/nwprod/hpc-stack/libs/hpc-stack/modulefiles/stack
+module use /scratch1/NCEPDEV/nems/Raffaele.Montuoro/dev/nasa/libs/hpc-stack/1.1.0-nasa/modulefiles/stack
```

Comment:
What's the path forward with not relying on Raffaele's stack? What's the timeline for getting what's needed in the emc hpc stack?

Reply:
These are also only for hera, so if we are doing this, shouldn't we be consistent throughout the platforms.

Reply:
Also in the strategy used here: https://github.com/ufs-community/ufs-weather-model/pull/591 only the modules that are different use the personal stack, perhaps that's a better way forward?  So only: 
module use -a /scratch1/NCEPDEV/nems/Raffaele.Montuoro/dev/nasa/ufs/lib/modulefiles

 module load gftl-shared/v1.3.0
 module load yafyaml/v0.5.1
 module load mapl/v2.7.1-beta

 module load intelpython/3.6.8

While all others use the hpc-stack from nwprod?